packages/grafana-e2e/cypress/plugins/index.js

```diff
@@ -12,6 +15,17 @@ module.exports = (on, config) => {
       return null;
     },
   });
+  on('task', {
+    getJSONFiles: async ({ dirPath }) => {
+      const directoryPath = path.join(__dirname, dirPath);
+      console.log(`Attempting to read contents of directory: ${directoryPath}`);
```

Comment:
is this console.log meant to be left here?

Reply:
I did because I thought asking clients of this lib to have to type in weird paths with `../../../forever` was tricky and they could use debugging help but I think now that I'm updating this so that you don't have to do that, I'll remove it!